ci-operator/step-registry/openshift-logging/patch-tlssecurityprofile-apiserver/openshift-logging-patch-tlssecurityprofile-apiserver-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -euo pipefail
+
+export KUBECONFIG="${SHARED_DIR}/kubeconfig"
```

Comment:
We need to set proxy to connect to private cluster, it's better to check if the proxy is needed or not, ref: https://github.com/openshift/release/blob/master/ci-operator/step-registry/enable-qe-catalogsource/enable-qe-catalogsource-commands.sh#L7

Reply:
Added